wrappers/services.go

```diff
@@ -705,6 +748,47 @@ func (es *ensureSnapServicesContext) ensureSnapSlices(quotaGroups *quota.QuotaGr
 	return nil
 }
 
+func (es *ensureSnapServicesContext) ensureSnapJournaldUnits(quotaGroups *quota.QuotaGroupSet) error {
+	handleJournalModification := func(grp *quota.Group, path string, content []byte) error {
+		old, modifiedFile, err := tryFileUpdate(path, content)
+		if err != nil {
+			return err
+		}
+
+		if modifiedFile {
```

Comment:
Nitpick^2, how about
```
if !modifiedFile {
  return nil
}```

Reply:
Good point! Thanks for your review here, really appreciated